public/locales/en/translations.json

```diff
@@ -57,15 +57,19 @@
       "field": "Field",
       "fieldPlaceholder": "Select field",
       "calculation": "Calculation",
-      "calculationPlaceholder": "select",
+      "calculationPlaceholder": "Select",
```

Comment:
Move the rulesFlyout translations to be under the page they go with.  So in this case.... under the rules, then under .flyouts.
See devices.flyouts.[flyoutname] for examples.

Reply:
fixed